dotnet/minimal-api/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG OTEL_EXPORTER_OTLP_ENDPOINT
+ARG OTEL_EXPORTER_OTLP_HEADERS
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+WORKDIR /app
+
+COPY *.csproj ./
+RUN dotnet restore
+
+COPY . ./
+RUN dotnet publish -c Release -o out
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-bullseye-slim
+WORKDIR /app
+COPY --from=build-env /app/out .
+ENV OTEL_EXPORTER_OTLP_ENDPOINT = ${OTEL_EXPORTER_OTLP_ENDPOINT}
+ENV OTEL_EXPORTER_OTLP_HEADERS = ${OTEL_EXPORTER_OTLP_HEADERS}
```

Comment:
Let's remove these `ENV` lines and instead just read the uesr's system environment variables in in `docker-compose.yml`

Reply:
I hesitate on this one in part because by leaving these lines we reinforce our need for the environment variables to be passed without causing harm/confusion.